projects/orangecrab-nix/cabal.project

```diff
@@ -0,0 +1,5 @@
+packages:
+  orangecrab.cabal
+
+write-ghc-environment-files: always
+
```

Comment:
```suggestion

-- Eliminates the need for `--enable-tests`, which is needed for HLS.
tests: true
```
(I just copied this from `simple-nix`. If you think it's incorrect, please correct `simple-nix`)